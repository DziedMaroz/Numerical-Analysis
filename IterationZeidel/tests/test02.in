3
4 1 -2 8
1 -5 1 -10
3 1 -5 10
1 1 1 
0.00000001

