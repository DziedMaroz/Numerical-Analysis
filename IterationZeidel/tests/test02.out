RUNNING ON TEST: tests/test02.in
SIMPLE ITERATION:
K = 35
-1.000 -2.000 1.000 
ZEIDEL:
K = 13
-1.000 -2.000 1.000 

